protoc_plugin/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 * Generate code comments for annotated protobuf inputs. ([#161])
 
+* New protoc_plugin flag `generate_constructor_arguments` added to bring back
+  the old message factory methods with arguments. These arguments were removed
+  in 21.0.0 ([#703]) as they caused bloat in generated binaries in some cases.
+
+  Example usage:
+
+  ```
+  protoc --dart_out='generate_constructor_arguments,<other options>:.' ...
```

Comment:
The flag is a bit long, I wonder if something shorter like `constructor_args` would be better?

Reply:
Yeah, shorter is probably better here.